app/controllers/api/v1/users_controller.rb

```diff
@@ -0,0 +1,30 @@
+class Api::V1::UsersController < ApplicationController
+    before_action :authorize_request, except: :create
+    before_action :find_user, except: %i[create index]
+
+    CUSTOMER = 'C'
```

Comment:
can you explain me what is this ? is the first time i have seen it

Reply:
Deleted